.jest.config.cjs

```
@@ -40,9 +40,9 @@ module.exports = {
         '<rootDir>/scripts/agents/includes/sync-version.js',
     ],
     moduleNameMapper: {
-        '^(\.{1,2}/.*)\.js$': '$1',
+        '^(\.{1,2}/.*?)\.(?:js|cjs)$': '$1',
```

ESLint

[error] Unnecessary escape character: \..

(no-useless-escape)